cdap-runtime-ext-dataproc/pom.xml

```diff
@@ -133,7 +133,17 @@
     </dependency>
     <dependency>
       <groupId>org.powermock</groupId>
-      <artifactId>powermock-api-mockito</artifactId>
+      <artifactId>powermock-api-mockito2</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>com.fasterxml.jackson.core</groupId>
+      <artifactId>jackson-databind</artifactId>
+      <scope>compile</scope>
```

Comment:
Why do we need to include it here and then exclude with HadoopClassExcluder? Can we just exclude it from the app file altogether?